vendor/engines/secure_rooms/app/services/secure_rooms/access_rules/account_selection_rule.rb

```diff
@@ -0,0 +1,19 @@
+module SecureRooms
+
+  module AccessRules
+
+    class AccountSelectionRule < BaseRule
+
+      def self.condition(_user, _card_reader, accounts, selected)
+        if selected.present?
+          Verdict.new(:grant)
+        elsif accounts.present? && selected.blank?
+          Verdict.new(:pending, "Must select Account")
+        end
```

Comment:
oh, we did actually get the official word from the client that one account = instant grant, right? I forgot